index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Setup</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <div id="name"><a href="#">John Doe</a></div>
```

Comment:
- [ ] You used the correct anchor tag for the logo, but the color of the logo does not match with the template. The `a` tag is a little bit tricky when using parent styles, so I recommend you use the `color: inherit` attribute or add the color directly to the logo `a` tag.
 - Your logo:
 - 
<img width="324" alt="image" src="https://user-images.githubusercontent.com/61919617/219476700-80a446eb-a9cc-416a-aee0-b7329db70a26.png">

 - Template:
 - 
<img width="359" alt="image" src="https://user-images.githubusercontent.com/61919617/219476865-344a67f5-d2fa-4811-96b5-e96c4fc57c05.png">